CHANGELOG.md

```diff
@@ -115,7 +115,7 @@ the [docs](https://docs.konghq.com/gateway/2.7.x/reference/external-plugins/).
   The new method is faster and uses less memory
   [#8204](https://github.com/Kong/kong/pull/8204)
 - Multiple improvements in the Router. Amongst others:
-  - The router builds twice as faster
+  - The router builds twice as fast compared to prior kong versions
```

Comment:
I think the change might be unnecessary.

Reply:
Just noting what I thought was proper english.

Reply:
ping @Kong/team-docs, PTAL

Reply:
Another option is to end the statement at “twice as fast” and leave the implication that it refers to prior kong versions.

Reply:
^ this last option is what we did for the CE/EE combined changelog: https://docs.konghq.com/gateway/changelog/#performance

I would go with that for simplicity.

Reply:
Looks like T got it handled.